skafka/src/main/scala/com/evolutiongaming/skafka/producer/Producer.scala

```diff
@@ -91,14 +91,14 @@ object Producer {
   }
 
   @deprecated("Use of(ProducerConfig)", since = "12.0.1")
-  def of[F[_]: ToTry: Async](
+  def of[F[_]: Async](
     config: ProducerConfig,
     executorBlocking: ExecutionContext
   ): Resource[F, Producer[F]] = {
     of(config)
   }
 
-  def of[F[_]: ToTry: Async](
+  def of[F[_]: Async](
```

Comment:
breaks backward compatibility

Reply:
I though it might break it, but MiMa checks passing kinda convinced me otherwise.

Reply:
Or do you mean source compatibility?